src/Tribe/Tickets_Handler.php

```diff
@@ -1079,6 +1102,32 @@ public function tickets_order_input( $post_id ) {
 	}
 
 	/**
+	 * Sorts tickets according to stored menu_order
+	 *
+	 * @since TBD
+	 *
+	 * @param array $tickets array of ticket objects
+	 *
+	 * @return array - sorted array of ticket objects
+	 */
+	public function sort_tickets_by_menu_order( $tickets ) {
+		// our sort function
+		function sortTicketsByMenuOrder( $a, $b ) {
```

Comment:
alter this name to be snake_cased — `_sort_tickets_by_menu_order_callback` or something